midend/lib/Conversion/OpSimplification/AddConstReshape2Const.cpp

```diff
@@ -0,0 +1,131 @@
+//====- AddConstReshape2Const.cpp----------------------------===//
```

Comment:
Please follow the 80-character rule for the header; other [file](https://github.com/buddy-compiler/buddy-mlir/blob/main/midend/lib/Conversion/LowerRVV/LowerRVVPass.cpp) can be referenced.